examples/protocol_specification_ex/gym.yaml

```diff
@@ -22,3 +22,15 @@ speech_acts:
 ct:AnyObject: |
   bytes any = 1;
 ...
+---
+entry: [reset]
+reply:
+  reset: [status]
+  status: [act, close, reset]
```

Comment:
Adding a `status` message field seems necessary here in order for us to be able to specify a proper dialogue.